event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamJdbcRepository.java

```diff
@@ -48,20 +50,25 @@ public void insert(final EventStream eventStream) {
     }
 
     @Override
-    public Stream<EventStream> getFeed(final long offset, final Link link,
-                                       final long pageSize, final Map<String, Object> params) {
+    public Stream<EventStream> getFeed(final String position,
+                                       final Direction direction,
+                                       final long pageSize,
+                                       final Map<String, Object> params) {
         try {
             final PreparedStatementWrapper ps = preparedStatementWrapperOf(SQL_PAGE);
             ps.setLong(1, pageSize);
-            ps.setLong(2, offset);
+            if (FixedPosition.HEAD.getPosition().equals(position)){
```

Comment:
This just needs to be removed? - If yes reply, but please don't make the code change.

Reply:
Will be refactored in the next commit